ComponentKit/Core/CKComponentViewAttribute.h

```diff
@@ -194,11 +187,8 @@ namespace CK {
 
       void performSetter(id object, SEL setter) const
       {
-        const CachedSetter &set = CachedSetterInvocation(object, setter);
-        T value = _value;
-        [set.invocation setArgument:&value atIndex:2];
-        [set.invocation setSelector:setter];
-        [set.invocation invokeWithTarget:object];
+        const auto setterIMP = (void (*)(id, SEL, T))[object methodForSelector:setter];
```

Comment:
Curious what happens here if I have a method signature that expects a const T &, and then you pass me T by value?